diff -up pyenchant-1.6.5/setup.py.disable-distribute-setup pyenchant-1.6.5/setup.py
--- pyenchant-1.6.5/setup.py.disable-distribute-setup	2013-08-06 09:08:02.620478114 +0200
+++ pyenchant-1.6.5/setup.py	2013-08-06 09:08:13.130744158 +0200
@@ -6,8 +6,8 @@
 #
 
 
-import distribute_setup
-distribute_setup.use_setuptools()
+#import distribute_setup
+#distribute_setup.use_setuptools()
 from setuptools import setup, find_packages, Extension
 from distutils.archive_util import make_archive
 
